spec/models/item_spec.rb

```diff
@@ -3,5 +3,19 @@
 RSpec.describe Item, type: :model do
   describe 'relationships' do
     it { should belong_to :supermarket }
+    it{should have_and_belong_to_many :customers}
```

Comment:
good job making sure to test your AR associations. As per my comment on the db migration, this is a different syntax than what we've been teaching. Still works, just abstracts away some of what's happening.